app/views/user_mailer/new_user_in_leader_area.html.erb

```diff
@@ -0,0 +1 @@
+<%= @new_user.name %>, has joined Please email them at <%= @user_user.email %> and welcome them.
```

Comment:
Please connect and collaborate with @hollomancer, @kylemh and @jenn (not sure of Jenn's GitHub handle) on this mailers:

- subject
- body